src/reducers/locations.js

```diff
@@ -11,6 +11,7 @@ import {
 } from '../actions';
 
 export const locationsReducer = (state = {}, action) => {
+  const dateString = (new Date()).toISOString();
```

Comment:
What is the `dateString` for?

Reply:
This is to update the metadata last_updated_date.